packages/auto-label/src/auto-label.ts

```diff
@@ -236,9 +212,7 @@ handler.addLabeltoRepoAndIssue = async function addLabeltoRepoAndIssue(
     repo
   );
 
-  let autoDetectedLabelAdded: string | undefined;
-
-  let wasAdded = false;
+  let wasNotAdded = true;
```

Comment:
I generally avoid negatives in variable names, especially with booleans.  It's possible this is fine, just saying it makes things harder to read.

Reply:
Yeah, I agree. However, it's ultimately what we want to count for, since we want to exit out of checking that individual repo if the label was not added x amount of times.